src/Hooks/CustomHooks.js

```diff
@@ -0,0 +1,18 @@
+import qs from 'query-string';
+import { useCallback } from 'react';
+
+export const useCreateUrlParams = (history) => {
```

Comment:
you have the exact same function `useCreateUrlParams` in MiscHelper.js.

Reply:
Yeah, this PR was not ready yet. You are fast ::)

Reply:
oups sorry :face_with_head_bandage: